src/vcpkg/portlint.cpp

```diff
@@ -0,0 +1,384 @@
+        {
+            handle_warning("vcpkg_configure_cmake", "vcpkg_cmake_configure");
+            if (fix == Fix::NO)
+            {
+                break;
+            }
+            const auto end = portfile_content.find(')', index);
+            const auto ninja = portfile_content.find("PREFER_NINJA", index);
+            if (ninja != std::string::npos && ninja < end)
+            {
+                const auto start = portfile_content.find_last_not_of(" \n\t\r", ninja - 1) + 1;
+                portfile_content.erase(start, (ninja - start) + StringLiteral("PREFER_NINJA").size());
+            }
+            portfile_content.replace(index, StringLiteral("vcpkg_configure_cmake").size(), "vcpkg_cmake_configure");
+            fixedPortfile.added_host_deps.insert("vcpkg-cmake");
+        }
+        if (Strings::contains(portfile_content, "vcpkg_build_cmake"))
+        {
+            handle_warning("vcpkg_build_cmake", "vcpkg_cmake_build");
+            if (fix == Fix::YES)
+            {
+                Strings::inplace_replace_all(portfile_content, "vcpkg_build_cmake", "vcpkg_cmake_build");
+                fixedPortfile.added_host_deps.insert("vcpkg-cmake");
+            }
+        }
+        if (Strings::contains(portfile_content, "vcpkg_install_cmake"))
+        {
+            handle_warning("vcpkg_install_cmake", "vcpkg_cmake_install");
+            if (fix == Fix::YES)
+            {
+                Strings::inplace_replace_all(portfile_content, "vcpkg_install_cmake", "vcpkg_cmake_install");
+                fixedPortfile.added_host_deps.insert("vcpkg-cmake");
+            }
+        }
+        index = 0;
+        while ((index = portfile_content.find("vcpkg_fixup_cmake_targets", index)) != std::string::npos)
+        {
+            handle_warning("vcpkg_fixup_cmake_targets", "vcpkg_fixup_cmake_targets");
+            if (fix == Fix::NO)
+            {
+                break;
+            }
+            const auto end = portfile_content.find(')', index);
+            const auto target = portfile_content.find("TARGET_PATH");
+            if (target != std::string::npos && target < end)
+            {
+                auto start_param = target + StringLiteral("TARGET_PATH").size();
+                start_param = portfile_content.find_first_not_of(" \n\t)", start_param);
+                const auto end_param = portfile_content.find_first_of(" \n\t)", start_param);
+                if (end_param != std::string::npos && end_param <= end)
+                {
+                    const auto original_param = portfile_content.substr(start_param, end_param - start_param);
+                    auto param = StringView(original_param);
+                    if (Strings::starts_with(param, "share/"))
+                    {
+                        param = param.substr(StringLiteral("share/").size());
+                    }
+                    if (param == "${PORT}" || param == origin)
```

Comment:
`share/<packname>`. `packname` can be lowercase or `packageCase`. @dg0yt  prefers lowercase to avoid adding extra dirs if portname = packagename. 

However the case for `packageCaseConfig.cmake` needs to be correct. While the `-config.cmake` version should always be lowercase.

Reply:
Yeah that is right. But it maybe break the portfile when it will replace some stuff in the files manually after they are moved. But I am fine with reverting the change